morphs/load/ephys.py

```diff
@@ -76,10 +68,30 @@ def ephys_data(block_path, good_clusters=None, collapse_endpoints=False, shuffle
     return spikes
 
 
-def _pickle(pickle_file):
-    try:
-        with open(pickle_file, 'rb') as f:
-            return pickle.load(f)
-    except UnicodeDecodeError as e:
-        with open(pickle_file, 'rb') as f:
-            return pickle.load(f, encoding='latin1')
+def ephys_data(block_path, good_clusters=None, collapse_endpoints=False, shuffle_endpoints=False, memoize=True):
```

Comment:
if not memoize and else could be if memoize and else...

Reply:
I guess I prefer smaller code block first for if else... although I could remove the else here...